actionpack/lib/action_dispatch/routing.rb

```diff
@@ -120,9 +120,9 @@ module ActionDispatch
   #
   #   # In config/routes.rb
   #   controller :blog do
-  #     get 'blog/show',    to: :list
```

Comment:
Can you remove those changes from this PR?

Reply:
Done. Sorry about the mistake.